client/src/store/MainStore.ts

```diff
@@ -1,33 +1,65 @@
-import { observable, action, runInAction } from 'mobx'
+import { observable, action } from "mobx";
+import { strict } from "assert";
 
 export interface IUser {
-    id: string
-    name: string
-    lastName: string
-    avatarUrl: string
-    isOnline: boolean
+  id: string;
+  name: string;
+  lastName: string;
+  avatarUrl: string;
+  isOnline: boolean;
 }
 
 export interface IMessage {
-    userId: string
-    text: string
-    timeStamp: string
+  userId: string;
+  fullName: string;
+  text: string;
+  timeStamp: string;
 }
 
+class MainStore {
+  @observable
+  public users: IUser[] = [];
 
-class MainStore {    
-    @observable
-    public users: IUser[] = []
+  @observable
+  public currentDialog: IMessage[] = [];
 
-    @observable
-    public currentDialog: IMessage[] = []
+  @observable
+  public clientId: string = "123";
 
-    @observable
-    public clientId: string = ''
-    
-    @action.bound
-    public getUsers() {
+  @observable
+  public newMessage: string = "";
 
-    }
+  constructor() {
+      this.getUsers()
+  }
+
+  @action.bound
+  public getUsers() {
+    const newUser = {
+      id: "35s4d354",
+      name: "Erjan",
+      lastName: "Egoev",
+      avatarUrl: "sldkjfls",
```

Comment:
you can copy an actual image orl from google

Reply:
ok